scripts/openstack-quickstart-demosetup

```diff
@@ -230,6 +236,12 @@ if [ "x$with_magnum" = "xyes" ]; then
     setup_keystone_authtoken /etc/magnum/magnum.conf magnum keystone_auth
 fi
 
+if [ "x$with_barbican" = "xyes" ]; then
+    setup_keystone_authtoken /etc/barbican/barbican.conf barbican
+    crudini --set /etc/barbican/barbican.conf keystone_authtoken auth_version 'v3.0'
+    sed -i 's/barbican_api/barbican-api-keystone/g' /etc/barbican/barbican-api-paste.ini
```

Comment:
Please use crudini and do what devstack is doing: https://github.com/openstack/barbican/blob/master/devstack/lib/barbican#L141